### DISCUSSION TITLE
Ignore End of File Differences

### DISCUSSION THREAD

**Discussion by pklorei** [WinMerge/winmerge#603] [Q&A]
Hi,

I am having an issue with file endings showing as different due to some whitespace issue.

![image](https://user-images.githubusercontent.com/52418878/106819288-9137da80-6647-11eb-9499-216eb516fc8c.png)

As I was searching on Google for an answer I saw a link to an old WinMerge forums question that seems to be same as mine. However that link is broken. 

[following is excerpt from google search results]
"Oct 15, 2020 · I have files whose only difference is the end of file: the 1st one ends with [CR][LF] when the 2nd one has no final [CR][LF] And I would like them to appear as identical"

This is a really significant issue as it affects most of my html and JS files (server vs GitHub comparison). Ignoring whitespace, blank lines, and carriage returns does not help.

**Comment by blommaep** [comment]
I have this on entire directories by difference of platform between project members. Very annoying as it renders Winmerge useless to identify the few files that really changed. So had to use a different diff/merge tool. I have all possible ignore flags on (ignore blanks, blank lines, linux/windows difference)
